src/components/progress/themes/circular/shared/circular.progress.bootstrap.scss

```diff
@@ -13,13 +13,13 @@ $theme: $bootstrap;
     font-weight: 700;
 }
 
-[part~='svg'][part~='indeterminate'] {
+[part='svg indeterminate']  {
```

Comment:
Are you sure that these are the only parts that can be applied to this element? As soon as any other part is applied the styles will stop working.